### ISSUE TITLE
[capitalized-comments] Should not autofix multiline when `ignoreConsecutiveComments`

### ISSUE THREAD

**Issue by IanVS** [eslint/eslint#7831]
**Tell us about your environment**

* **ESLint Version:** 3.12.2
* **Node Version:** 6.9.1
* **npm Version:** Yarn 0.18.1

**What parser (default, Babel-ESLint, etc.) are you using?** babel-eslint

**Please show your full configuration:**

```js
rules: {
  'capitalized-comments': [2, 'always', {'ignoreConsecutiveComments': true}],
}
```

**What did you do? Please include the actual source code causing the issue.**

Ran `eslint --fix` against code like:

```js
// this is a multi-line comment that does not start with a capital letter, and maybe
// describes some variables like:
// foo - this is a thing
// fooFighter - it fights the foo
```

**What did you expect to happen?**
```js
// This is a multi-line comment that does not start with a capital letter, and maybe
// describes some variables like:
// foo - this is a thing
// fooFighter - it fights the foo
```

**What actually happened? Please include the actual, raw output from ESLint.**
```js
// This is a multi-line comment that does not start with a capital letter, and maybe
// Describes some variables like:
// Foo - this is a thing
// FooFighter - it fights the foo
```

I'm proposing that when `ignoreConsecutiveComments` is `true`, the rule should not auto-capitalize lines other than the first line.

**Comment by platinumazure** [comment]
Weird- it shouldn't even report those other lines, so I can't understand why they're being autofixed in the first place.

**Comment by IanVS** [comment]
It won't report them if the _first_ line is capitalized.  It _does_ report them if the first line is lowercase.

**Comment by platinumazure** [comment]
Fascinating. I think that might be a bug in reporting- we should probably
tolerate those consecutive comments even if the first comment violates the
rule.

Thanks for the extra info, @IanVS.

**Comment by platinumazure** [comment]
I'd like to fix this, if possible. Hoping to get to it this weekend (certainly long before the next minor release next Friday).

**Comment by platinumazure** [comment]
This was fixed by #7835, I just forgot to add the "(fixes #7831)" to the commit message. Oops 😞
